extra/html-extra/HtmlExtension.php

```diff
@@ -36,7 +36,7 @@ public function getFilters(): array
     public function getFunctions(): array
     {
         return [
-            new TwigFunction('html_classes', [self::class, 'htmlClasses']),
+            new TwigFunction('html_classes', self::class::htmlClasses(...)),
```

Comment:
There is many changes like this:
```suggestion
            new TwigFunction('html_classes', self::htmlClasses(...)),
```